petastorm/tests/test_common.py

```diff
@@ -229,6 +229,7 @@ def expected_row(i):
             StructField('id_div_700', IntegerType(), False),
         ] + maybe_int_fixed_size_list_field +
         [
+            StructField('nested_struct', StructType([StructField('nested_int', IntegerType(), True)])),
```

Comment:
Sneak in a nested field that is considered currently unsupported and see that all tests still pass.